data/templates/default.typst

```diff
@@ -149,7 +149,7 @@ $elseif(bibliographystyle)$
 $endif$
 $if(bibliography)$
 
-#bibliography($for(bibliography)$"$bibliography$"$sep$,$endfor$$if(full-bibliography)$, full: true$endif$)
+#bibliography(($for(bibliography)$"$bibliography$"$sep$,$endfor$)$if(full-bibliography)$, full: true$endif$)
```

Comment:
Is this still going to work for the case where there is just one entry?
Don't you need `("mybib",)` rather than `("mybib")` for it to be treated as an array?

Reply:
Double checking this.